### PR TITLE
清理非代码文件

### PR DESCRIPTION
**Pull request by ylht**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,7 +22,12 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+# idea kotlin module
+*.kotlin_module
+*.kotlin_builtins
+
```
Comment (superhail) [inline .gitignore:28]: 这里为啥不直接ignore target这个dir

File: .gitignore [hunk 2]
```diff
@@ -22,7 +22,12 @@
...
 .idea
 TouchstoneDeployTool.iml
 
 .DS_Store
```
Comment (superhail) [inline .gitignore:32]: 这里应该要写成**/.DS_Store吧，因为我看好像有多个不同的地方都出现这个文件了

**Comment by superhail** [comment]
还有我注意到我们的包名好像都大写了，但是按照规范应该还是小写比较好。
[Package names are written in all lower case to avoid conflict with the names of classes or interfaces.](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html#:~:text=Package%20names%20are%20written%20in,names%20of%20classes%20or%20interfaces.&text=Name%20collisions%20that%20occur%20within,example.)

File: .gitignore (hunk 1 above)
Comment (ylht) [reply .gitignore:28]: Ok 学到了

File: .gitignore (hunk 2 above)
Comment (ylht) [reply .gitignore:32]: 嗯

Comment (ylht) [reply .gitignore:32]: 我试了下 不需要前缀，git会ignore所有的同名文件，anywhere

Comment (superhail) [reply .gitignore:32]: 那就好

File: .gitignore (hunk 1 above)
Comment (ylht) [reply .gitignore]: 改好了

**Comment by ylht** [comment]
> 还有我注意到我们的包名好像都大写了，但是按照规范应该还是小写比较好。
> [Package names are written in all lower case to avoid conflict with the names of classes or interfaces.](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html#:~:text=Package%20names%20are%20written%20in,names%20of%20classes%20or%20interfaces.&text=Name%20collisions%20that%20occur%20within,example.)

这个放到下一次pr解决吧

**Comment by superhail** [comment]
> > 还有我注意到我们的包名好像都大写了，但是按照规范应该还是小写比较好。
> > [Package names are written in all lower case to avoid conflict with the names of classes or interfaces.](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html#:~:text=Package%20names%20are%20written%20in,names%20of%20classes%20or%20interfaces.&text=Name%20collisions%20that%20occur%20within,example.)
> 
> 这个放到下一次pr解决吧

ok

**Pull request merged**